rpm/BUILD.bazel

```diff
@@ -47,6 +53,23 @@ bazeldnf(
     rulename = "libvirt-libs_aarch64",
 )
 
+bazeldnf(
+    name = "ldd_s390x",
+    command = "ldd",
+    libs = [
+        "/usr/lib64/libvirt-lxc.so.0",
+        "/usr/lib64/libvirt-qemu.so.0",
+        "/usr/lib64/libvirt.so.0",
+        "/usr/lib64/libgcrypt.so.20",
+        "/usr/lib64/libkrb5support.so.0",
+        "/usr/lib64/libkeyutils.so.1",
+        "/usr/lib64/liblz4.so.1",
+        "/usr/lib64/libmount.so.1",
+    ],
+    rpmtree = ":libvirt-devel_s390x",
+    rulename = "libvirt-libs_s390x",
+)
+
```

Comment:
Minor detail, but I think that these changes to `rpm/BUILD.bazel` belong to the next commit.

Reply:
No, they are needed for `make rpm-deps` to work. Specifically it is needed for:
```
# update tar2files targets which act as an adapter between rpms
    # and cc_library which we need for virt-launcher and virt-handler
    bazel run \
        --config=${ARCHITECTURE} \
        //rpm:ldd_s390x
```
Though i think for actual merging i might squash all the `rpm-deps` commits into one. I just didn't for now because it makes rebasing much easier if i can just run `make rpm-deps` instead of solving merge conflicts in that huge file...